matpower/matpower-model/src/main/java/com/powsybl/matpower/model/MatpowerReader.java

```diff
@@ -145,17 +157,19 @@ private static void readGenerators(Matrix generators, MatpowerModel model) {
             gen.setStatus(generators.getInt(row, 7));
             gen.setMaximumRealPowerOutput(generators.getDouble(row, 8));
             gen.setMinimumRealPowerOutput(generators.getDouble(row, 9));
-            gen.setPc1(generators.getDouble(row, 10));
-            gen.setPc2(generators.getDouble(row, 11));
-            gen.setQc1Min(generators.getDouble(row, 12));
-            gen.setQc1Max(generators.getDouble(row, 13));
-            gen.setQc2Min(generators.getDouble(row, 14));
-            gen.setQc2Max(generators.getDouble(row, 15));
-            gen.setRampAgc(generators.getDouble(row, 16));
-            gen.setRampTenMinutes(generators.getDouble(row, 17));
-            gen.setRampThirtyMinutes(generators.getDouble(row, 18));
-            gen.setRampQ(generators.getDouble(row, 19));
-            gen.setApf(generators.getDouble(row, 20));
+            if (generators.getDimensions()[1] > MATPOWER_V1_GENERATORS_COLUMNS) {
```

Comment:
As discussed together, it would be safer to use the following test here:
`if (generators.getDimensions()[1] >= MATPOWER_V2_GENERATORS_COLUMNS)`

=> 

- if dimension < `MATPOWER_V1_GENERATORS_COLUMNS`: error
- if dimension in `[ MATPOWER_V1_GENERATORS_COLUMNS ; MATPOWER_V2_GENERATORS_COLUMNS [`: considered as V1 format;
- if dimension >= `MATPOWER_V2_GENERATORS_COLUMNS`: considered as V2 format.